### PR TITLE
Fix status page to be awesome.

### PR DESCRIPTION
**Pull request by applmak**
This PR significantly changes the status page to be more functional and useful. It provides far more information than the old status page, and provides a new control surface.

Critically, the status page is now served from a new server, not the normal wall server. In the future, this might move entirely outside of this npm itself.

### CODE REVIEW

File: bin/run_1x1.sh [hunk 1]
```diff
@@ -32,4 +33,17 @@ DEBUG=wall:* NODE_PATH='.:node_modules' node \
...
-  "$@"
+  "$@" &
+BRICK_PID="$!"
+
+# Run status server:
+./status/run_status.sh &
+STATUS_PID="$!"
+
+function clean_up {
+  kill $BRICK_PID $STATUS_PID;
```
Comment (jmegq) [inline bin/run_1x1.sh]: double-quotes around each $*_PID please.

File: bin/run_1x1.sh [hunk 2]
```diff
@@ -32,4 +33,17 @@ DEBUG=wall:* NODE_PATH='.:node_modules' node \
   --module_dir 'demo_modules/*' \
   --use_geometry '[{"right":1},{"down":1},{"left":1},{"up":1}]' \
   --assets_dir demo_assets \
-  "$@"
+  "$@" &
+BRICK_PID="$!"
```
Comment (jmegq) [inline bin/run_1x1.sh]: readonly BRICK_PID=...

File: bin/run_1x1.sh [hunk 3]
```diff
@@ -32,4 +33,17 @@ DEBUG=wall:* NODE_PATH='.:node_modules' node \
...
+
+# Run status server:
+./status/run_status.sh &
+STATUS_PID="$!"
```
Comment (jmegq) [inline bin/run_1x1.sh]: readonly STATUS_PID=...

File: server/util/log.js [hunk 4]
```diff
@@ -13,15 +13,19 @@ See the License for the specific language governing permissions and
...
-var pushError = function(record) {
+export const recentErrors = [];
+const ERROR_BUFFER_SIZE = 100;
+
+function pushError(record) {
   recentErrors.push(record);
-  if (recentErrors.length > ERROR_BUFFER_SIZE) {
+  emitter.emit('error', record);
+  while (recentErrors.length > ERROR_BUFFER_SIZE) {
     recentErrors.shift();
```
Comment (jmegq) [inline server/util/log.js]: optional: emit a log that we "dropped N errors" when you do this?

File: bin/run_1x1.sh (hunk 2 above)
Comment (applmak) [reply bin/run_1x1.sh]: Done

File: bin/run_1x1.sh (hunk 3 above)
Comment (applmak) [reply bin/run_1x1.sh]: Done.

File: bin/run_1x1.sh (hunk 1 above)
Comment (applmak) [reply bin/run_1x1.sh]: Done.

File: server/util/log.js (hunk 4 above)
Comment (applmak) [reply server/util/log.js]: It will always be one with the current implementation, so not worth it.

**Pull request merged**